config/errors.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2023 The KBase Project and its Contributors
+// Copyright (c) 2023 Cohere Consulting, LLC
```

Comment:
has it really been a year since this project was started?! it doesn't feel like it...